test/fuzz/fuzz-unit-file/directives.mount

```diff
@@ -143,6 +143,7 @@ ReadWritePaths=
 RemoveIPC=
 RestartKillSignal=
 RestrictAddressFamilies=
+RestrictFileSystems=
```

Comment:
merge this commit that adds to test/fuzz into the commit that adds the directive - otherwise those tests that check for this will fail until this commit